client-v2/src/services/editionsApi.ts

```diff
@@ -71,3 +71,13 @@ export const getIssueSummary = async (id: string): Promise<EditionsIssue> => {
     return response.json();
   });
 };
+
+export const publishIssue = async (id: string): Promise<EditionsIssue> => {
+  return pandaFetch(`/editions-api/issues/${id}/publish`, {
+    method: 'POST',
+    credentials: 'same-origin'
+  }).then(response => {
+    // TODO publish endpoint returns a 204 No Content, we probably want something...
+    return response.json();
```

Comment:
perhaps the newly published hash (based on our discussions about tracking changes)? regardless - this certainly doesn't return an `EditionsIssue`.